src/table/components/PaginationContent.tsx

```diff
@@ -70,19 +70,26 @@ function PaginationContent({
   const showRowsPerPage = !isSelectionMode && shouldShow('rppOptions', width) && totalColumnCount <= 100;
   const displayedRowsText = translator.get('SNTable.Pagination.DisplayedRowsLabel', [
     `${page * rowsPerPage + 1} - ${Math.min((page + 1) * rowsPerPage, totalRowCount)}`,
-    totalRowCount,
+    totalRowCount.toString(),
```

Comment:
Isn't is a number ?

Reply:
yes

Reply:
.. so why have you converted it to string ?

Reply:
because translator wants strings

Reply:
Ok, I see.